### PR TITLE
Replace treeview bmp with ico

### PR DESCRIPTION
**Pull request by ozone10**
- use recommended subclassing
- fix missing icons in function list when searching

ref #14959

### CODE REVIEW

**Review by donho — CHANGES_REQUESTED** [review]
~~Remove SetWindowSubclass call for not modifying the retured value and keep the legacy windows subclass way.~~

**Comment by ozone10** [comment]
rebased to fix conflict with master

File: PowerEditor/src/NppBigSwitch.cpp [hunk 1]
```diff
@@ -3670,7 +3670,7 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			_subDocTab.setFont();
 			_mainDocTab.resizeIconsDpi();
 			_subDocTab.resizeIconsDpi();
-			_subDocTab.setCloseBtnImageList();
+			_mainDocTab.setCloseBtnImageList();
```
Comment (donho) [inline PowerEditor/src/NppBigSwitch.cpp:3673]: 👍

File: PowerEditor/src/WinControls/TreeView/TreeView.h [hunk 2]
```diff
@@ -119,16 +116,13 @@ class TreeView : public Window {
 	bool searchLeafAndBuildTree(const TreeView & tree2Build, const std::wstring & text2Search, int index2Search);
 	void sort(HTREEITEM hTreeItem, bool isRecusive);
 	void customSorting(HTREEITEM hTreeItem, PFNTVCOMPARE sortingCallbackFunc, LPARAM lParam, bool isRecursive);
-	BOOL setImageList(int w, int h, int nbImage, int image_id, ...);
+	bool setImageList(std::vector<int> imageIds, int imgSize = 0);
```
Comment (donho) [inline PowerEditor/src/WinControls/TreeView/TreeView.h:119]: 👍

**Comment by donho** [comment]
> rebased to fix conflict with master

No worry, I'm doing it locally.

**Pull request closed without merging**